cmd/configure.go

```diff
@@ -73,14 +75,19 @@ var openVPNConfigCmd = &cobra.Command{
 	},
 }
 
-// openVPNConfigureCmd represents the `furyagent configure openvpn` command
+// openVPNClientConfigureCmd represents the `furyagent configure openvpn-client` command
 var openVPNClientConfigCmd = &cobra.Command{
 	Use:   "openvpn-client",
-	Short: "Get OpenVPN users client from s3",
+	Short: "Create and revocate OpenVPN users",
```

Comment:
Create and revoke (is `revocate`an italian word?

Reply:
It's not an italian word, but definitely it looks like it. 😅 

Fixed.